dist/backdraft.js

```diff
@@ -1214,42 +1257,67 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
```

Comment:
Wouldn't be better to create separate Views for the filters and encapsulate templates and events in there? There's something similar in ActionTable when setting up ActionBar filters.
As for the `window.activeMenu` variable, that seems to be too intrusive. I would store it in the table instead.